src/ResourceManager/Sql/Commands.Sql/help/New-AzureRmSqlElasticPool.md

```diff
@@ -214,7 +214,6 @@ The acceptable values for this parameter are:
 Type: String
 Parameter Sets: DtuBasedPool
 Aliases:
-Accepted values: None, Premium, Basic, Standard, DataWarehouse, Stretch, Free, PremiumRS
```

Comment:
Unsure.
The edition should be using these valid editions:

        [PSArgumentCompleter("None",
            Management.Sql.Models.DatabaseEdition.Basic,
            Management.Sql.Models.DatabaseEdition.Standard,
            Management.Sql.Models.DatabaseEdition.Premium,
            Management.Sql.Models.DatabaseEdition.DataWarehouse,
            Management.Sql.Models.DatabaseEdition.Free,
            Management.Sql.Models.DatabaseEdition.Stretch,
            "GeneralPurpose", "BusinessCritical")]

Looks like this should updated anyway since GeneralPurpose and BusinessCritical needs to be added and PremiumRS needs to be deleted.

Will update these markdown files manually.

@maddieclayton for markdown generation, do PSArgumentCompleter values get automatically generated or do they get excluded?

Reply:
Unlike ValidateSet or using an enum, it looks like PSArgumentCompleter doesn't generate the list of strings provided.
I'll add manually.

Reply:
Updated to the correct accepted values @maddieclayton

Reply:
Yes that's correct, PSArgumentCompleter is not caught by platyPS (it was created by us while ValidateSet is a built in powershell attribute)